src/_includes/layouts/base.njk

```diff
@@ -29,10 +29,14 @@
 
 		{% include "footer.njk" %}
 
-		<script async src="{{ '/' | url }}js/main.min.js?v={{getCurrentVersion.version}}"></script>
-    <script src="{{ '/' | url }}js/details-element-polyfill/details-element-polyfill.polyfill.min.js"></script>
-		<script src="{{ '/' | url }}js/table-of-contents/table-of-contents.min.js"></script>
-		<script src="{{ '/' | url }}js/contact.min.js"></script>
+    <script async src="{{ '/js/details-element-polyfill/details-element-polyfill.polyfill.min.js' | url }}"></script>
+		{% if page.url === '/checklist/' %}
+		<script src="{{ '/js/checklist.min.js?v=' + getCurrentVersion.version | url }}"></script>
+		{% endif %}
+		{% if page.url === '/contact/' %}
```

Comment:
Any reason for not using `elif` instead of `if`. I don't think in any situation both the if conditions can match.

Reply:
Nothing special. As a developer, I just tend not to write control flows with `else if` – especially not when cases are mutually exclusive, like this